python/graphstorm/model/node_gnn.py

```diff
@@ -155,6 +155,7 @@ def node_mini_batch_gnn_predict(model, loader, return_label=False):
     -------
     Tensor : GNN prediction results.
     Tensor : GNN embeddings.
+    Tensor : Node index corresponding to prediction results.
```

Comment:
Do we need to also change edge_gnn.py to add Edge index as a return value?

Reply:
it does not return node index actually, I forgot to remove the comment.